lib/utils/string.ts

```diff
@@ -29,21 +29,21 @@ export const toCamelCase = (string: string): string =>
   string.toLowerCase().replace(/((?:-|_|\s)\w)/g, (k: string) => k[1].toUpperCase());
 
 /**
- * Convert the string to PascalCase
+ * Converts the first character in the string to an uppercase equivalent.
  * @param string
  */
-export const toPascalCase = (string: string): string =>
-  capitalize(string.toLowerCase().replace(/((?:-|_|\s)\w)/g, (k: string) => k[1].toUpperCase()));
+export const capitalize = (string: string): string => string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
The _capitalise_ word denotes that the _entire_ string is changed to uppercase, however, that is not the case in here. Could we rename the function to something like `firstLetterToUpperCase()` or `firstToUpperCase()`?

Reply:
Same here. `.capitalize()` methods are well-known for this usage in programming languages, so I will keep this word.

Reply:
Well, I still find it ambiguous, as _capitalisation_ means to _capitalise all letters in one or more words_, while what this function do is actually to keep the casing of all letters but the first letter of the first word is changed to upper-case, which is usually called _sentence case_.

[Dictionary.com](https://www.dictionary.com/browse/capitalise) says that _capitalise_ means _to write or print in capital letters letters or with an initial capital letter_, which confirms that your prefered sense is valid (I didn’t say it is not), however, it also confirms that its usage is ambiguous (it means either capitalisation of _all letters_ or _the initial/first letter_).

Reply:
Yes, but it remains a common practice. So I'm going to keep what developers are used to seeing and using  :man_shrugging:.